server/channels/app/post_metadata.go

```diff
@@ -169,13 +171,17 @@ func (a *App) getEmbedsAndImages(c request.CTX, post *model.Post, isNewPost bool
 		post.Metadata = &model.PostMetadata{}
 	}
 
-	// Embeds and image dimensions
-	firstLink, images := a.getFirstLinkAndImages(post.Message)
-
 	if post.Metadata.Embeds == nil {
 		post.Metadata.Embeds = []*model.PostEmbed{}
 	}
 
+	if post.GetProp(UnsafeLinksPostProp) != nil {
```

Comment:
Looks like if I set `UnsafeLinksPostProps` to "false" is going to generate problems. Shouldn't be it checking both, the `nil` and the `"true"` string.

Reply:
Yes. I shouldn't have used "true" without making sure "false" doesn't work. I just wanted to check to see if the prop was set at all instead of checking for a specific value. 
But if this confused you, it's going to confuse someone else so I will change it so the prop has to be "true".